.github/workflows/ci-workflow.yml

```diff
@@ -63,6 +63,7 @@ jobs:
         COLLECTOR_LICENSE: ${{ secrets.COLLECTOR_LICENSE }}
         LASP_LICENSE: ${{ secrets.LASP_LICENSE }}
         LASP_SECURE_LICENSE: ${{ secrets.LASP_SECURE_LICENSE }}
+        FORCE_RUN_TESTS_WITH_SECRETS: ${{ github.repository_owner == 'newrelic' && github.head_ref == 'master' }}
```

Comment:
Guessing the specific branch check is to avoid running in PR's from forked repos, since ENV vars aren't limited to the main repo like secrets.

Reply:
Ya, ran into an issue where the repository_owner for PRs from forked repos is still `newrelic`. And some people work from the master branch on their forked repo. So had to add both checks.